src/core/mesh/qgsmeshtimesettings.h

```diff
@@ -72,7 +72,7 @@ class CORE_EXPORT QgsMeshTimeSettings
   private:
 
     QString mRelativeTimeFormat = QStringLiteral( "d hh:mm:ss" );
-    QString mAbsoluteTimeFormat = QStringLiteral( "dd.MM.yyyy hh:mm:ss" );
+    QString mAbsoluteTimeFormat = QStringLiteral( "yyyy-MM-dd HH:mm:ss" );
```

Comment:
why this change? this causes to fail travis test.

Reply:
to accord date time format with other part of QGIS. This had to be done for a moment...